en/console-and-shells.rst

```diff
@@ -354,6 +354,19 @@ By default, a ``requested`` extra param is automatically added when a Shell
 is dispatched using ``dispatchShell()``. This ``requested`` parameter prevents
 the CakePHP console welcome message from being displayed on dispatched shells.
 
+Exit Status Codes
+-----------------
+It can sometimes be useful to return exit status codes. Simply use php's ``exit()`` statement.
```

Comment:
Hm, but then false and true really feel wired compared to the exit codes (like exactly the opposite).
If direct exit statements create problems then adding a wrapper ($this->exit()) is an option (that will not execute exit() when testing)? Also return true/false don't allow returning OTHER exit codes but 1 and thus make shell script communication harder than not using cake shell environments (in this single regard), not the case?